pelops/features/hog.py

```diff
@@ -8,10 +8,11 @@
 
 class HOGFeatureProducer(FeatureProducer):
 
-    def __init__(self, chip_producer, image_size=(256,256), cells=(16, 16), orientations=8):
+    def __init__(self, chip_producer, image_size=(256,256), cells=(16, 16), orientations=8, histogram_bins_per_channel=256):
```

Comment:
This is an excellent question that I don't have a good answer to. In theory there is no reason to limit ourselves to the same size, but perhaps we want to so that the scaling effects are the same for both HOG and ResNet. That seems like an OK argument, so I'll change it.